### ISSUE TITLE
Module not found manspider

### ISSUE THREAD

**Issue by Barriuso** [blacklanternsecurity/MANSPIDER#7]
I just installed using pipx and it does not work.

![image](https://user-images.githubusercontent.com/16231048/124608139-c3f03400-de6e-11eb-88b8-c2e73bf3a3fa.png)

Waiting for your response.

**Comment by RackunSec** [comment]
Hello, @Barriuso 

I am not a developer, but I got it working without `pipx`. I did:

```bash
git clone https://github.com/blacklanternsecurity/MANSPIDER
cd MANSPIDER
python3 -m pip install -r requirements.txt
```

Then edit the file `/path/to/repo/manspider.py` and remove the period before `lib` on line 8.

```bash
python3 manspider.py --help
```

**Comment by TheTechromancer** [comment]
Fixed in 2019070f3233263eb19d932ffcffa39e37c3f137.
